src/plugins/share/server/url_service/saved_objects/register_url_service_saved_object_type.ts

```diff
@@ -20,6 +21,7 @@ export const registerUrlServiceSavedObjectType = (
 ) => {
   const urlSavedObjectType: SavedObjectsType = {
     name: 'url',
+    indexPattern: ANALYTICS_SAVED_OBJECT_INDEX,
```

Comment:
Does it have to by analytics index? Short URLs are use by analytics apps, but they don't have to be use only by Analytics apps, they can be used by other teams, too.

Reply:
It doesn't have to, no.

Now, there was a mail from Rudolf to all contributors with a TTL of April, 19 (which will be yesterday) for feedback regarding the reallocation / where types should be moved.

Now that this TTL is passed, I'm not sure we still have the time to revisit it without putting the delivery of the feature even more at risk to be honest.

cc @rudolf @gsoldevila

Reply:
Discussed about that on slack. We will note such reallocation tweaks and perform them as fixes after the merge

I'll add a section to the issue's description for that

Reply:
I think it's a valid point, but as Pierre mentioned we can't spend a lot of time moving types around given our deadline.
I will create a PR with the requested change, to trigger CI and check for potential impact.
If there's no impact, we can attempt to merge it with the feature branch before FF.
Otherwise, updating this as a fix (post FF) seems like a viable path too.